contents/blog/scarf-open-source-projects.md

```diff
@@ -0,0 +1,51 @@
+date: 2022-01-28
+title: Why open-source projects are essential for large businesses
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+keywords: 
+featuredImage: ../images/blog/series-b/series-b-baby.png
+featuredImageType: full
+author: ["joe-martin"]
```

Comment:
Author should be Avi.

Reply:
Yup, it's just a draft atm. Need to sort adding a new contributor etc first ;)